### PR TITLE
Make Travis green

### PR DESCRIPTION
**Pull request by mvidner**
I noticed the Travis builds have been failing because of some setup fragility: https://travis-ci.org/YorickPeterse/ruby-lint/builds 
So I made a few tries at fixing that and had success:
Here's a build of 12 successful jobs: https://travis-ci.org/mvidner/ruby-lint/builds/132864909

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+  - gem install bundler
+
 script: bundle exec rake
 
 rvm:
   - 1.9.3
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.5
```
Comment (YorickPeterse) [inline .travis.yml]: Doesn't `2.2` automatically pick the latest patch level version?

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
   - rbx-2
-  - jruby
+  - jruby-9.0.5.0
```
Comment (YorickPeterse) [inline .travis.yml]: Same here, I believe `jruby` is supposed to automatically use the latest 9k version. I'd rather not update the Travis config file every time a new Ruby version comes out.

File: .travis.yml (hunk 1 above)
Comment (mvidner) [inline .travis.yml]: No, in fact it picks 2.2.0. I haven't checked why.

File: .travis.yml (hunk 2 above)
Comment (mvidner) [inline .travis.yml]: Before, this was picking jruby-1.7.something.

I admit that the Travis/RVM versioning logic is a mystery to me. In a project of mine where I try to use multiple rubies ([ruby-dbus](https://travis-ci.org/mvidner/ruby-dbus)) I care about the result being green for correct code, so I simply marked the less stable rubies that I don't care as much about as allowed failures:
```yml
matrix:
  allow_failures:
    - rvm: ruby-head
    - rvm: rbx
```

**Comment by YorickPeterse** [comment]
Thanks!

**Pull request merged**